### DISCUSSION TITLE
OpenFeature SDK for angular

### DISCUSSION THREAD

**Discussion by dinbtechit** [open-feature/community#373] [Ideas]
Angular has a lot of similarities with nestjs. I see there is a dedicated SDK for NestJS. I would like to use that as an inspiration and create one for angular. But would like to know if anyone have attempted this and would like to give me any suggestion.

Cheers!

**Comment by beeme1mr** [comment]
Hey @dinbtechit, not yet but it's something we would like to do. It would be amazing if you were able to get one started. I know @toddbaert had some ideas for how it could work. Perhaps we could collaborate on an issue in the JS repo.

FYI @lukas-reining @thomaspoignant

**Reply by lukas-reining** [reply]
Yes @dinbtechit we talked about this several times but had no one really start it yet.
I guess the Nest.JS SDK will be quite similar to something in Angular due to DI, but will have to provide the Client specific parts, e.g. will use the [static paradigm](https://openfeature.dev/specification/glossary/#sdk-paradigms) of the [client SDK](https://github.com/open-feature/js-sdk/tree/main/packages/client).
As said, opening an issue in the [JS SDK repo](https://github.com/open-feature/js-sdk) to discuss any questions and make visible your work would be great!

**Reply by dinbtechit** [reply]
Thank you - @lukas-reining and @beeme1mr for the input. I'll open issues in the JS SDK repo if I have any questions! Cheers!

**Reply by thomaspoignant** [reply]
I agree that it could be great to have something for angular.

**Comment by KarlangaXZ** [comment]
:heart:

**Comment by rkrbaddam** [comment]
Regarding the Angular SDK, is there any code reference we can use for Angular apps that don't utilize the Angular SDK?

**Reply by beeme1mr** [reply]
Hey @rkrbaddam, you should be able to use the [web sdk](https://openfeature.dev/docs/reference/technologies/client/web/) in Angular.

**Comment by lukas-reining** [comment]
We have an experimental Angular SDK now :)
https://www.npmjs.com/package/@openfeature/angular-sdk
